EventFilter/SiPixelRawToDigi/plugins/SiPixelFedCablingMapGPU.cc

```diff
@@ -25,8 +25,8 @@ void processCablingMap(SiPixelFedCablingMap const& cablingMap, SiPixelFedCabling
   std::vector<unsigned int>  RawId(MAX_SIZE);
   std::vector<unsigned int>  rocInDet(MAX_SIZE);
   std::vector<unsigned int>  moduleId(MAX_SIZE);
-  std::vector<unsigned char> badRocs(MAX_SIZE);
-  std::vector<unsigned char> modToUnp(MAX_SIZE);
+  std::vector<short int>     badRocs(MAX_SIZE);
+  std::vector<short int>     modToUnp(MAX_SIZE);
```

Comment:
why `short int` ?
these are always `bool` in the code

Reply:
I have already tried with bool but it did not work